giraffe/src/utils/tooltip.ts

```diff
@@ -38,6 +47,25 @@ const getTooltipGroupColumns = (
     }
   })
 
+const getTooltipYCumulativeCol = (
+  table: Table,
+  lineData: LineData,
+  rowIndices: number[],
+  yColKey: string,
+  getValueFormatter: (colKey: string) => (x: any) => string,
+  rowColors: string[] | null
+): TooltipColumn => {
+  const yColData = getDomainDataFromLines(lineData, 'ys')
+  const yFormatter = getValueFormatter(yColKey)
+  return {
+    key: yColKey,
+    name: 'cumulative',
+    type: table.getColumnType(yColKey),
+    colors: rowColors,
+    values: rowIndices.map(i => yFormatter(yColData[i])),
+  }
+}
```

Comment:
some jest tests for this would be fantastic

Reply:
Will add the tests with the next push.